portfolio/pom.xml

```diff
@@ -33,6 +33,16 @@
       <artifactId>appengine-api-1.0-sdk</artifactId>
       <version>1.9.59</version>
     </dependency>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
+    <dependency>
```

Comment:
Duplicate dep?

Reply:
Actually this is *double* duplicated. It's already present before the change

Was this added by some tool?